server/models/operation.js

```diff
@@ -22,6 +22,7 @@ let Operation = americano.getModel('bankoperation', {
     categoryId: String,
     attachments: Object,         // {linkTranslationKey: String, linkPlainEnglish: String, url: String}
     operationTypeID: String,
+    customLabel: String,
```

Comment:
Don't forget to also transfer the customLabel on merge! (server side and client side)